Ginger/GingerCoreNET/RunLib/CLILib/CLIOptions.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Reflection;
+using CommandLine;
+using GingerCoreNET.SourceControl;
+
+namespace Amdocs.Ginger.CoreNET.RunLib.CLILib
+{
+
+    class CLIOptionClassHelper
+    {
+        public const string FILENAME = "filename";
+
+        public static string GetClassVerb<T>()
+        {
+            VerbAttribute verbAttr = typeof(T).GetCustomAttribute<VerbAttribute>();
+            return verbAttr.Name;
+        }
+
+        public static string GetAttrShorName<T>(string name)
+        {
+            OptionAttribute attrOption = typeof(T).GetProperty(name).GetCustomAttribute<OptionAttribute>();
+            return attrOption.ShortName;
+        }
+    }
+
+    public class OptionsBase  // ' Base class for Options whihc execute run set
+    {
+        public enum eVerboseLevel
+        {
+            normal,
+            debug            
+        }
+
+        [Option('v', "verbose", Required = false,Default = eVerboseLevel.normal,  HelpText = "Select Verbose level: normal, debug")]
+        public eVerboseLevel VerboseLevel { get; set; }
+    }
+
+        
+
+    [Verb("example", HelpText = "Show example")]
```

Comment:
not better to break it down to separate class files under same folder?

Reply:
Done